pkg/util/grpc/encoding/snappy/snappy.go

```diff
@@ -0,0 +1,81 @@
+package snappy
+
+import (
+	"io"
+	"sync"
+
+	"github.com/golang/snappy"
+	"google.golang.org/grpc/encoding"
+)
+
+// Name is the name registered for the snappy compressor.
+const Name = "snappy"
+
+func init() {
+	encoding.RegisterCompressor(newCompressor())
+}
+
+type compressor struct {
```

Comment:
It would be nice to have small unit test for `compressor` type? WDYT?

Reply:
I've added tests.